prog/vnet/update_load_balancer_node.rb

```diff
@@ -60,7 +63,10 @@ def generate_lb_based_nat_rules
     end
 
     ipv6_prerouting = if load_balancer.ipv6_enabled?
-      load_balancer.active_vm_ports.uniq { |row| row.load_balancer_port.id }.map do |vm_port|
+      load_balancer.active_vm_ports { |ds| ds.eager(:load_balancer_port) }
+        .uniq(&:load_balancer_port_id)
+        .sort_by { |vm_port| vm_port.load_balancer_port.src_port }
```

Comment:
This is the same code as in the IPv4 branch.  Please move the shared code above the `if`, and assign it to a local variable.  Use the local variable in both the IPv4 and IPv6 branches.